molgenis-semantic-mapper/src/main/java/org/molgenis/data/mapper/exception/IncompatibleDataTypeException.java

```diff
@@ -0,0 +1,66 @@
+package org.molgenis.data.mapper.exception;
+
+import org.molgenis.data.meta.AttributeType;
+
+import java.text.MessageFormat;
+
+import static java.lang.String.format;
+import static java.util.Objects.requireNonNull;
+import static org.molgenis.data.i18n.LanguageServiceHolder.getLanguageService;
+
+public class IncompatibleDataTypeException extends IncompatibleTargetException
+{
+	private static final String ERROR_CODE = "M05";
+	private final String mappingTargetAttributeName;
+	private final AttributeType mappingTargetAttributeType;
+	private final String targetRepositoryAttributeName;
+	private final AttributeType targetRepositoryAttributeType;
+
+	public IncompatibleDataTypeException(String mappingTargetAttributeName, AttributeType mappingTargetAttributeType,
```

Comment:
Decide if we want to use localized attribute labels. In that case, pass Attribute instead of strings

Reply:
Why would we not, if everything is translated, why should this be any different